package.json

```
@@ -12,16 +12,19 @@
 	"license": "Apache-2.0",
 	"private": true,
 	"scripts": {
+		"pretest": "yarn build",
 		"test": "jest --collect-coverage",
-		"test:ci": "jest --verbose --ci --no-stack-trace --collect-coverage",
+		"test:ci": "jest --verbose --ci --runInBand --no-stack-trace --collect-coverage",
 		"docs": "typedoc --name discord.js --exclude **/index.ts --exclude **/*.test.ts --includeVersion --json docs/docs.json",
-		"prebuild": "npm run clean",
+		"prebuild": "yarn clean",
 		"build": "lerna run build",
-		"clean": "npx rimraf packages/*/dist",
-		"lint": "eslint packages --ext .ts",
-		"lint:fix": "eslint packages --ext .ts --fix",
-		"prettier": "prettier --write **/*.{ts,js,json,yml,yaml}",
-		"postinstall": "npx lerna bootstrap"
+		"clean": "rimraf packages/**/dist",
+		"lint": "eslint packages --ext mjs,js,ts",
+		"lint:fix": "eslint packages --ext mjs,js,ts --fix",
+		"format": "prettier --write **/*.{ts,js,json,yml,yaml}",
+		"update": "yarn upgrade-interactive --latest",
+		"prepare": "husky install",
+		"postinstall": "lerna bootstrap"
```

```suggestion
		"prepare": "husky install"
```

You don't need `lerna bootstrap` when using yarn workspaces. Yarn will handle all linking for you, including when package A depends on package B. Example: `@sapphire/time-utilities` depends on `@sapphire/utilities`, which works without bootstrap just fine, as long as the "package A" is build because "main" and "types" will of course reference files in "dist", which doesn't exist otherwise.